ci/pipeline.yml

```diff
@@ -30,7 +31,7 @@ resources:
     initial_version: 1.3.1
     file: version
 
-- name: ci-develop
```

Comment:
@lucaspinto  this should not be committed in the final version

Reply:
Yes, I am doing some temporary changes to test the docs generation pipeline.

Reply:
OK NP, just to be sure, I thought the PR was almost ready.